web/src/js/ducks/settings.js

```diff
@@ -0,0 +1,92 @@
+import { StoreCmds } from '../actions'
+import { addLogEntry } from './eventLog'
+
+export const WS_MSG_TYPE = 'settings'
+export const WS_MSG_CMD_RESET = 'reset'
+export const WS_MSG_CMD_UPDATE = 'update'
```

Comment:
This seems duplicate. Can we share the same constants for both events emitted on the python end and events emitted in JS? Happy to help you with the Python part. :smiley: